src/Dfc.CourseDirectory.Web/Controllers/CourseSummaryController.cs

```diff
@@ -118,45 +118,20 @@ public IActionResult Index(Guid? courseId, Guid? courseRunId)
                 }
             }
 
-            if(courseRun.Regions != null)
+            if (courseRun.Regions != null)
             {
-                var allRegions = _courseService.GetRegions().RegionItems;
-                var regions = GetRegions().RegionItems.Select(x => x.Id);
-                vm.Regions = FormattedRegionsByIds(allRegions, courseRun.Regions);
+                var allRegions = _courseService.GetRegions();
+
+                vm.Regions = allRegions.RegionItems
```

Comment:
I'm finding this linq pretty hard to parse in terms of intent. Could you figure out a way to make the intent clearer?

Reply:
Updated as discussed.